test/inductor/test_cpp_wrapper.py

```diff
@@ -179,6 +179,10 @@ class BaseTest(NamedTuple):
         BaseTest("test_relu"),  # multiple inputs
         BaseTest("test_repeat_interleave", "", test_cpu_repro.CPUReproTests()),
         BaseTest("test_scalar_input"),
+        BaseTest("test_scatter4"),
+        BaseTest("test_scatter5"),
+        BaseTest("test_scatter6"),
+        BaseTest("test_scatter_reduce2"),
```

Comment:
Are we adding all `test_scatter` related tests?

Reply:
Scatter UTs have been added.